Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -197,7 +197,7 @@ - (void)waitForAbsenceOfView;
         KIFTestWaitCondition(found.view, error, @"Cannot find view containing accessibility element \"%@\"", found.element);
         
         // Hidden views count as absent
-        KIFTestWaitCondition([found.view isHidden] || [found.view superview] == nil, error, @"Accessibility element \"%@\" is visible and not hidden.", found);
+        KIFTestWaitCondition([found.view isHidden] || [found.view superview] == nil || ![found.view isPossibleVisibleInWindow], error, @"Accessibility element \"%@\" is visible and not hidden.", found);
```

Comment:
It feels odd doing this in `waitForAbsenceOfView`. Would this same view be usable for `waitForView` or `tap` actions? Potentially not the latter, due to maybe failing the `mustBeTappable:YES` param.

Reply:
FYI - not outdated, just marked that way due to the method name change.

Reply:
> Would this same view be usable for waitForView or tap actions?

From what I followed the code path it doesn't seem like it's being used in `waitForView` or `tap`